themes/prism-material-dark.css

```diff
@@ -0,0 +1,220 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  text-align: left;
+  white-space: pre;
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
+  hyphens: none;
+  color: inherit;
```

Comment:
I find this very strange. You use `inherit` here but for both `:not(pre) > code[class*="language-"]` and `pre[class*="language-"]`, you use `#eee`. Why not set it here?

Same for `background`.

Also, same for the two other themes as well.

Reply:
You're right, I'll modify that.